src/Selenium2Library/keywords/_alert.py

```diff
@@ -7,7 +8,9 @@
 class _AlertKeywords(KeywordGroup):
 
     def __init__(self):
-        self._cancel_on_next_confirmation = False
+        self._accept = 'accept'
+        self._dismiss = 'dismiss'

```

Comment:
Are these constants? If yes, they could be class variables and use capital letters:

```python
class _AlertKeywords(...):
    _ACCEPT = 'accept'
    _DISMISS = 'dismiss'
```